app/models/main.py

```diff
@@ -77,23 +77,6 @@ def remove_personal_data(self):
         """
         pass
 
-    @staticmethod
-    def validate_personal_data_removed(mapper, connection, instance):
```

Comment:
Rather than moving the validator definition out of the mixin, can we leave the function here but only add the listener on the relevant class?

Reply:
So I did have a think about this. But I think it's kind of misleading to have it on the mixin if it's not applied to everything.
The method now technically only applies to User objects. It's whole purpose has changed, from validating all personal data models to validating only user objects. That's because we've decided that objects that have had their personal data removed _can_ be changed but Users cannot be changed once their personal data is removed. 
In the future it's possible that this will be re implemented but I think, although it's a nifty bit of code, it'll just be a confusing relic for people to read. Plus I think the code represents the decision as is.

Reply:
Fair point